src/discord-gateway.c

```diff
@@ -688,8 +701,8 @@ _discord_gateway_session_from_json(struct discord_gateway_session *session,
         int url_len = (int)f->v.len;
 
         url_len = snprintf(session->base_url, sizeof(session->base_url),
```

Comment:
Okay, this part here looks good, since it's gonna get OOB-checked after the fact.